### ISSUE TITLE
Can't set artifact name

### ISSUE THREAD

**Issue by bpappin** [dcendents/android-maven-gradle-plugin#48]
the documentation says to use rootProject.name = 'artifact' to set the artifact name, but it's not working properly. It seems to be ignored by the plugin.

**Comment by bpappin** [comment]
FYI its using the project name instead.

**Comment by dcendents** [comment]
This needs to be set in settings.gradle and not in build.gradle.
Are you doing that?

I have an integration test that does that and it is working.

**Comment by bpappin** [comment]
The settings.gradle applies to the entire project, so if I have multiple libraries, they will all have the same artifact name.

Should I put in a new bug for that problem?

**Comment by dcendents** [comment]
Yes settings.gradle applies to the whole project, but you need to set the name of each project individually int it.

``` gradle
include 'app'
include 'library'
include 'baseLibrary'
include 'util'

rootProject.name = 'newName'
project(':library').name = 'newLibName'
```

But then you need to use the new name in build.gradle.
e.g.: app will link to `compile project(':newLibName')`

another option to simply generate a different artifact name but not change the gradle project name is to set the property `project.archivesBaseName = 'newLibName'` in build.gradle.
